backend/pkg/boards/main.go

```diff
@@ -0,0 +1,154 @@
+package boards
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/network/tftp"
+	"time"
+)
+
+const (
+	AckId      = "1"
+	DownloadId = "2"
+	UploadId   = "3"
+
+	BlcuOrderId = "blcu order"
+
+	DownloadName = "download"
+	UploadName   = "upload"
+
+	DSuccess = "download success"
+	USuccess = "upload success"
+)
+
+type Boards struct {
+	boardId  string
+	BoardAPI abstraction.BoardAPI
+	TempData []byte
+	AckChan  chan struct{}
+}
+
+func New(Id string) *Boards {
+	return &Boards{
+		boardId: Id,
+		AckChan: make(chan struct{}),
+	}
+}
+func (boards *Boards) Id() string {
+	return boards.boardId
+}
+
+func (boards *Boards) Notify(notification abstraction.BoardNotification) {
+	switch notification.Event() {
+	case AckId:
+		boards.AckChan <- struct{}{}
+
+	case DownloadId:
+		err := boards.BoardAPI.SendMessage(abstraction.TransportMessage(
+			&BlcuPing{
+				ID: BlcuOrderId,
+			}))
```

Comment:
The message you need to send is defined in `transport/packet/data`. You need to call `data.NewPacketWithValues(id, values)` with the appropiate ID and then in values what we sent last year was a value with the board to be programmed (this year might change, leave a placeholder with a TODO).